src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -738,10 +738,10 @@ class PubNub internal constructor(
      *
      * State information is supplied as a JSON object of key/value pairs.
      *
-     * If [PNConfiguration.sendStateWithHeartbeat] is `true`, the state for channels will be saved in the PubNub
-     * client and resent with every heartbeat. In that case, it's not recommended to mix setting state
+     * If [PNConfiguration.maintainPresenceState] is `true`, the state for channels will be saved in the PubNub
+     * client and resent with every heartbeat and resubscribe. In that case, it's not recommended to mix setting state
      * through channels *and* channel groups, as state set through the channel group will be overwritten
-     * after the next heartbeat.
+     * after the next heartbeat or resubscribe.
```

Comment:
Is "resubscribe" the same as "subscribe reconnection"?

I am asking because in PNConfiguration there is:  
 "for the current [userId] with every automatic heartbeat (if [heartbeatInterval] is greater than 0) and subscribe reconnection."

Reply:
hmm what's the best way to describe the underlying subscribe calls (the automatic ones that happen every 300 seconds, not necessarily in response to any user facing api like reconnect() or subscribe())

Reply:
maybe just "subscribe connection" ? without going into details?

Reply:
LGTM.